learn/guides/distinctive-language-features/data.md

```diff
@@ -52,7 +53,7 @@ A decimal number represents a decimal fraction exactly. That means that the prec
 
 You can use a literal character ``d`` to indicate a ``decimal`` value, and similarly ``f`` for a ``float`` value.
 
-Decimal numbers are not the same as binary floating point numbers defined as per IEEE standard. They are not built into the ALU logic on the processor. Therefore their computation is slower yet more accurate and precise.
+Decimal numbers are not the same as binary floating-point numbers defined as per IEEE standard. They are not built into the ALU logic on the processor. Therefore their computation is slower yet more accurate and precise.
```

Comment:
I believe we've decided not to use hyphen here. @praneesha, can you please confirm?

Reply:
+1